package.json

```diff
@@ -56,6 +56,7 @@
   "dependencies": {
     "@floating-ui/react": "^0.24.1",
     "classnames": "^2.3.2",
+    "fathom-client": "^3.5.0",
```

Comment:
Please move to `devDependencies` and keep in mind that anything you add to `dependencies` becomes bundled with every user's `npm i flowbite-react`, including production deployments for their app which does not include any of our documentation